lib/skycore/payload_builder.rb

```diff
@@ -36,6 +36,7 @@ def build_save_mms(text, fallback_text, attachments)
         x.API_KEY api_key
         x.NAME "tatango_test"
         x.FALLBACKSMSTEXT fallback_text
+        x.SUBJECT subject if subject
```

Comment:
According to Derek, `subject` is required, so the `if` shouldn't be there. Also, I'm not familiar with the Skycore API, but shouldn't `SUBJECT` be next to the `TEXT`?